library-backend/library-api/src/main/java/org/internship/library/api/BookService.java

```diff
@@ -0,0 +1,10 @@
+package org.internship.library.api;
+
+public interface BookService {
+
+    Book getBook(String id);
+    Book createBook(Book book);
+    Book updateBook(String id);
```

Comment:
We should update the book with something, right? Similar to the create.

Only having the id here makes us know which book we'll update but we won't know with what new book we'll update it.

Reply:
You're right :D